src/Components/AuthInput/AuthInput.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { KeyboardTypeOptions, Text, TextInput, View } from 'react-native';
+
+import styles from './styles';
+
+interface AuthInputProps {
+  input: string;
+  setInput: React.Dispatch<React.SetStateAction<string>>;
+  defaultValue: string;
+  isPassword: boolean;
+  displayInput: boolean;
+  setDisplayInput: React.Dispatch<React.SetStateAction<boolean>>;
+  keyboard: KeyboardTypeOptions;
+  autoCap: boolean;
+}
+
+export default function AuthInput({
+  input,
+  setInput,
+  defaultValue,
+  isPassword,
+  displayInput,
+  setDisplayInput,
+  keyboard,
+  autoCap,
+}: AuthInputProps) {
+  const [placeholder, setPlaceholder] = useState<string>(defaultValue);
```

Comment:
Remove this and instead pass down `placeholder` and `setPlaceholder` as props. This will fix the bug